static/app/components/workflowEngine/gridCell/actionCell.tsx

```diff
@@ -1,69 +1,16 @@
-import styled from '@emotion/styled';
-
-import {Flex} from 'sentry/components/container/flex';
-import {Tooltip} from 'sentry/components/core/tooltip';
-import {IconCircledNumber} from 'sentry/components/iconCircledNumber';
 import {EmptyCell} from 'sentry/components/workflowEngine/gridCell/emptyCell';
-import {ActionMetadata} from 'sentry/components/workflowEngine/ui/actionMetadata';
-import {space} from 'sentry/styles/space';
 import type {ActionType} from 'sentry/types/workflowEngine/actions';
+import {AutomationActionSummary} from 'sentry/views/automations/components/automationActionSummary';
 
 type ActionCellProps = {
   actions: ActionType[];
   disabled?: boolean;
 };
 
-export function ActionCell({actions, disabled}: ActionCellProps) {
+export function ActionCell({actions}: ActionCellProps) {
   if (!actions || actions.length === 0) {
     return <EmptyCell />;
   }
 
-  if (actions.length === 1 && actions[0]) {
-    const metadata = ActionMetadata[actions[0]];
-    if (!metadata) {
-      return <EmptyCell />;
-    }
-    const {name, icon} = metadata;
-    return (
-      <Flex align="center" gap={space(0.75)}>
-        <IconContainer>{icon}</IconContainer>
-        {name}
-      </Flex>
-    );
-  }
-  const actionsList = actions
-    .map(action => ActionMetadata[action]?.name)
-    .filter(x => x)
-    .join(', ');
-  return (
-    <ActionContainer align="center" gap={space(0.75)}>
-      <IconContainer>
-        <IconCircledNumber number={actions.length} />
-      </IconContainer>
-      <Tooltip title={actionsList} disabled={disabled}>
-        <ActionsList>{actionsList}</ActionsList>
-      </Tooltip>
-    </ActionContainer>
-  );
+  return <AutomationActionSummary actions={actions} hasTooltip />;
```

Comment:
Extracted this out into a different component because we need to also render outside of a table cell